lang/python/python3-notify2/Makefile

```diff
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-notify2
+PKG_VERSION:=0.3.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=notify2
+PKG_HASH:=33fa108d50c42f3cd3407cc437518ad3f6225d1bb237011f16393c9dd3ce197d
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-notify2
+	SECTION:=lang
+	CATEGORY:=Languages
+	SUBMENU:=Python
+	TITLE:=Python interface to DBus notifications
+	DEPENDS:=+python3 +python3-dbus
+endef
```

Comment:
```suggestion
define Package/python3-notify2
  SECTION:=lang
  CATEGORY:=Languages
  SUBMENU:=Python
  TITLE:=Python interface to DBus notifications
  URL:=https://bitbucket.org/takluyver/pynotify2
  DEPENDS:=+python3 +python3-dbus
endef
```

Reply:
Is it compatible with pynotify2?

Reply:
URL added. I need to investigate if desktop-notify is compatible... It does essentially the same, but since I don't use it for my code, it's dependency for another package, so some changes might be required there...